app/javascript/chat/userDetails.jsx

```diff
@@ -89,12 +89,21 @@ export default class UserDetails extends Component {
     }
     let blockButton = '';
     if (channel.channel_type === 'direct' && window.currentUser.id != user.id) {
-      blockButton = setUpButton({ 
-          modalId: 'userdetails__blockmsg',
-          otherModalId: 'userdetails__reportabuse',
-          btnName: 'Block User'
+      blockButton = setUpButton({
+        modalId: 'userdetails__blockmsg',
+        otherModalId: 'userdetails__reportabuse',
+        btnName: 'Block User',
       });
     }
+
+    let reportButton = '';
+    // if (window.currentUser.id !== user.id) {
```

Comment:
shouldn't this be part of the code?

Reply:
Yes, it should. I forgot to uncomment it

Reply:
can you add a test for it as well? Thanks!